tests/main/generic-classic-reg/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: |
```

Comment:
generic-classic-serial might be a more clear name here (I can see us looking for *serial* under tests).

Reply:
so far all our device registration tests end in -reg